apps/storybook/stories/config.ts

```diff
@@ -47,6 +47,7 @@ const demoIndexes: DemoIndexConfig = {
   },
   recipes: {
     open: true,
+    disableAi: true,
```

Comment:
just to be even more clear, I would probabily rename this as `disableGenerativeAI` or similar, so it's clear which 'ai' we refer to